ecosystem/sep-0024.md

```diff
@@ -190,6 +192,73 @@ The deposit flow can only be fulfilled if the Stellar `Account` has established
+
+**Anchors**: It's optional for anchors to use submit Claimable Balances as deposit transactions to Stellar accounts that don't yet have a trust-line to the asset they have requested.
+
+**Wallet Claimable Balance Flow**
+
+Wallets will still follow the [typical deposit flow](#Basic-Wallet-Implementation).
+The POST request however will extend to include a `claimable_balance_supported` parameter
+
+**Wallet Claimable Balance Polling**
+
+Claimable Balance Polling allows the wallet to update the claimant that their is a balance available to claim and they may need to establish[trust-line](#stellar-account-doesnt-trust-asset) in order to do so.
+There are a few ways a wallet can support .
+
+1. The wallet can poll Horizon's [`/claimable_balances/`](https://developers.stellar.org/api/resources/claimablebalances/list/) list endpoint to pull any available claimable balances for the claimant.
+
+   * *Recommended request* `GET /claimable_balances?claimant={:claimant}&order={asc,desc}&limit={1-200}` or [See List Claimable Balances](https://developers.stellar.org/api/resources/claimablebalances/list/)
+
+1. Wallets could decode the `results_xdr` of the transaction submitted
+   1. Traverse each transaction's `results_xdr` to if it was submitted as a claimable balance
+   2. then pull the `claimable_balance_id` from the `xdr`
+   3. Use the `claimable_balance_id` to query Horizon's [`/claimable_balances/`](https://developers.stellar.org/api/resources/claimablebalances/single/) single endpoint to pull all the information needed for the claimable balance.
+
+   * *Recommended request* `GET /claimable_balances/:claimable_balance_id`
+
+   * Example of how a wallet could decode the `results_xdr`
+
+    ```js
+    const StellarBase = require('stellar-base');
+
+    const { xdr } = StellarBase;
+
+    const b64 = "AAAAAAAAAGQAAAAAAAAAAQAAAAAAAAAOAAAAAAAAAAA9/w3pAlg0fEoZp+eC9g0slbYtvoySN3i5y+F4WJY8sgAAAAA=";
+    const tr = xdr.TransactionResult.fromXDR(b64, 'base64');
+
+    const operationIndex = 0; // if you have multiple operations then you the index to correspond to he claimable balance operation
+    console.log(tr.result().results()[operationIndex].value().createClaimableBalanceResult().balanceId().toXDR('hex'));
+     ```
+
+**Wallet Claim Account Claimable Balance**
+
+It's recommended that Wallets have an feature that allows the account to claim the balances that were polled from Horizon.
+[Example on how to claim a balance](https://developers.stellar.org/docs/glossary/claimable-balance/#claim-claimable-balance)
+
+**Anchor Claimable Balance Flow**
+
+1. If a user's Stellar account does not have a trust-line to the asset's issuer account, one must be added via the [Change Trust](https://developers.stellar.org/docs/start/list-of-operations/#claim-claimable-balance) operation.
+2. The `Anchor` proceeds with the deposit flow via the [CAP-0023](../core/cap-0023.md) Create Claimable Balance operation.
```

Comment:
Is this correct? Why would an anchor have to add a change_trust op to the transaction? In fact, I don't think this _can_ happen because the user's SK is required to change the user account's trustlines